mkdocs.yml

```diff
@@ -57,6 +57,26 @@ plugins:
     # See: https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin
     - awesome-pages:
         filename: .nav.yml
+    - llmstxt:
+        markdown_description: > 
+          A Pack is an extension that adds new powers to your doc.
+          What other platforms call plugins, add-ons, or extensions, we call "Packs". 
+          A Pack extends the core building blocks of Coda, allowing users to build more powerful and integrated docs. 
+          Anyone can build a Pack, and they can use it in their own docs, share it with their team, or publish it to the world.
+          Packs are built using JavaScript or TypeScript, either on your local machine or in the browser, and run on Coda's own servers. 
+          We have a gallery where users can discover and install Packs, as well as a robust set of admin controls.
+        full_output: llms-full.txt
```

Comment:
Is this output file meant to be checked in, or does it only get generated by the release tooling?

Reply:
Yes, the file needs a preamble about what the documentation is about. This config file is hand-written.

Reply:
Ah, I see. Both of those files are generated by this MkDocs plugin during the doc build process. Like the other doc artifacts, we currently don't check these into version control, but generate them on-the-fly as part of the build and deploy process.